README.md

```diff
@@ -6,7 +6,7 @@
 
 ## Settings overview
 
-The image displays with minimal padding by default or can be configured to fill the navigation bar height. For mobile, an alternative, mobile-friendly image can be specified or the logo can be hidden entirely.
+The image displays with minimal padding by default or can be configured to fill the navigation bar height. For mobile, an alternative, mobile-friendly image can be specified or the logo can be hidden entirely. The logo can be shown on the menu, on specific pages, or everywhere.
```

Comment:
Perhaps add the default display to the last line.

e.g. The logo can be shown on the menu, on specific pages, or everywhere as default.

Reply:
Done in e62e714